packages/babel-helper-remap-async-to-generator/src/index.js

```diff
@@ -14,11 +14,24 @@ let buildWrapper = template(`
   })
 `);
 
+let arrowBuildWrapper =  template(`
+  (() => {
+    var ref = FUNCTION;
+    return (PARAMS) => ref.apply(this, arguments);
```

Comment:
Or we can run certain tests before publish or not as often etc. https://phabricator.babeljs.io/T2765#72428 yeah we probably need an evaluated test for this